catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/SortedQueryMonitorTest.java

```diff
@@ -215,6 +228,106 @@ public void interruptThirdFuture() throws Exception {
             NullPointerException.class, InterruptedException.class, InterruptedException.class);
   }
 
+  @Test
+  public void testSorts() throws Exception {
+    PropertyName propertyName = mock(PropertyName.class);
+    when(propertyName.getPropertyName()).thenReturn(TEST_PROPERTY);
+
+    SortBy ascendingSort = mock(SortBy.class);
+    when(ascendingSort.getSortOrder()).thenReturn(SortOrder.ASCENDING);
+    when(ascendingSort.getPropertyName()).thenReturn(propertyName);
+
+    SortBy descendingSort = mock(SortBy.class);
+    when(descendingSort.getSortOrder()).thenReturn(SortOrder.DESCENDING);
+    when(descendingSort.getPropertyName()).thenReturn(propertyName);
+
+    testSorting(new String[] {null, "a"}, new String[] {null, "a"}, ascendingSort);
```

Comment:
✏️  Should we split these 10 unit test cases in 10 separate unit test cases?

Reply:
Fixed.